tools/scripts/run_vulkan_tests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# run_vulkan_tests - Runs tests against Vulkan device
```

Comment:
Run tests that should work with the Vulkan device driver.

Reply:
> * VERY experimental, few features supported
> * requires google's `clspv` for compilation
> * documentation provided in `doc/sphinx/source/vulkan.rst`
> * tests can be run with `tools/scripts/run_vulkan_tests`



> * VERY experimental, few features supported
> * requires google's `clspv` for compilation
> * documentation provided in `doc/sphinx/source/vulkan.rst`
> * tests can be run with `tools/scripts/run_vulkan_tests`

nice! any comparison vs clvk would be nice..